src/altinn-app-frontend/src/utils/databindings.test.ts

```diff
@@ -111,6 +112,12 @@ describe('utils/databindings.ts', () => {
     ];
   });
 
+  describe('getKeyIndex', () => {
+    it('should work', () => {
```

Comment:
Would suggest a more descriptive title 🙂

f.ex like this? Or something else if you can come up with something 😁
```suggestion
    it('should return key indexes from string', () => {
```

Reply:
Right, I agree! I blame the rush.. ;-) Fixing this later.